mlbstatsapi/mlb_api.py

```diff
@@ -168,21 +258,55 @@ def get_people_id(self, fullname, sport_id: int = 1,
                     continue
         return player_ids
 
-    def get_teams(self, sport_id: int = 1, **params) -> List[Team]:
+    def get_teams(self, **params) -> List[Team]:
```

Comment:
since sport is a parameter let's put it up here or move it to other parameters

Reply:
Agree